lua/cfc_prop_restoration/core/sv/sv_init.lua

```diff
@@ -137,100 +177,112 @@ local function notifyOnError( ply )
     end
 end
 
-local function getPropVelocities( props )
-    local velocities = {}
 
+local function storePropVelocities( props, propCount )
     if not props then return {} end
```

Comment:
This function no longer returns the velocities table, so this should just return nothing to save allocating a table for no reason
```suggestion
    if not props then return end
```